index.js

```diff
@@ -30,7 +30,12 @@ var resUser = module.exports = function (chatId, callback) {
         json: true
     }, function (error, response, body) {
         if (!error && response.statusCode == 200) {
-            callback(error, body && body.result)
+            var result = body && body.result;
+            if(result)callback(error, result);
+            else callback(new Error('Username not found'));
+        }
+        else{
+            callback(error)
```

Comment:
`error` is **not** necessarily guaranteed to be truthy! Why? At line 32, we are checking if **error is not truthy** and **response status code equals 200**! The `else` will be triggered when error is **not** truthy but response status code does **not** equal 200! Thus the `callback(error)` may find that `error` is `null` (falsey).

I believe there's a better way to structure the code in the callback (line 32-39) to make it more robust and comprehensible.